src/core/blockchain/zilliqa/config.ts

```diff
@@ -172,7 +172,8 @@ export const config: IBlockchainConfig = {
             service: BlockchainNameService.CNS,
             record: ['crypto.ZIL.address']
         }
-    ]
+    ],
+    amountToKeepInAccount: new BigNumber(Math.pow(10, 14))
```

Comment:
JS is not reliable for big numbers, Math.pow uses native js numbers so it's not safe. use `BigNumber.pow` or use a string value.